workers.rb

```diff
@@ -308,9 +310,8 @@ def find_paper(issue_id)
     return paper_paths.first
   end
 
-  def download(issue_id, clear_cache)
-    FileUtils.rm_rf("tmp/#{issue_id}") if Dir.exist?("tmp/#{issue_id}") if clear_cache
-    Open3.capture3("whedon download #{issue_id}")
+  def download(issue_id)
+    Open3.capture3("whedon download #{issue_id} #{jid}")
```

Comment:
This is an example of using the change to the Whedon gem whereby the path for operating in can be passed as an argument.